app/src/forms/rbac/controller.js

```diff
@@ -140,4 +140,12 @@ module.exports = {
       next(error);
     }
   },
+  isUserPartOfFormTeams: async (req, res, next) => {
+    try {
+      const response = await service.isUserPartOfFormTeams(req.query);
```

Comment:
see comment for service. I think it's ok if you want to return a boolean to put the logic here, get the results from the service, then do your check for returning true/false. easier to maintain than having such a specific call in the service class.

Reply:
fixed